dataland-csvconverter/pdfMapping.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
The connection between the reports and the dataset is yet to be made, right? 
So after running this script we have the mapping between the dataset and the connected reports but still have to add this information to the dataset. Or have i missed something?

Reply:
I will document the process in the wiki once I get the time. The idea is the following: running this script collects the pdfs in the folder and it creates the output csv file. The content of that file is copy-pasted into the data dictionary. From there the csv is exported again as input for the other already existing bash script that will create the data for the container and push it. No need to do anything with the data sets.

Reply:
Maybe we should keep this open as a reminder until the documentation is updated.

Reply:
I updated the documentation in section "Real-Data-Container". Can you have a look and tell me if it is clear?

Reply:
Ok